### PR TITLE
Fix the links to the customizations section of component for deeplinks

### LINKED ISSUE #7888: Deep-linking on the developer.microsoft.com OUIFR site doesn't work.
Copy+paste+sending a deep link to a page/subpage on the fabric website renders a blank page. For example, visit https://developer.microsoft.com/en-us/fabric, select Components --> Colors --> Themes --> Output, copy the URL in the address bar (https://developer.microsoft.com/en-us/fabric#/customizations/colors#Output), open a new tab, and navigate to that URL.

#### Actual behavior:
 
A blank page is rendered:
image
 
#### Expected behavior:
 
The page manually navigated to is rendered:
image

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? **Yes** (but I don't have a ton of time right now, and this issue is a little embarrassing)
 
Requested priority: **Normal**
 
Products/sites affected: https://developer.microsoft.com/en-us/fabric
 
-----

As a separate issue, it's worth noting that the site still has a fairly large border on the left when rendered in a small window (see image under **Expected behavior**)

### PR DESCRIPTION
**Pull request by kenotron**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #7888
- [x] Include a change request file using `$ npm run change`

#### Description of changes

The customizations section isn't actually a top level section. As it is defined, these are under "components", so the logic fails. This PR corrects it.

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/7894)

### CODE REVIEW

File: apps/fabric-website/src/components/App/AppState.tsx [hunk 1]
```diff
@@ -792,7 +792,7 @@ export const AppState: IAppState = {
         },
         {
           title: 'Customization',
-          url: '#/components/customization',
+          url: '#/components/customizations',
```
Comment (natalieethell) [inline apps/fabric-website/src/components/App/AppState.tsx]: Just curious, why don't we change this to '#/customizations' instead?

Comment (kenotron) [reply apps/fabric-website/src/components/App/AppState.tsx]: So I'd have to make some structural changes to this that I'm not entirely comfortable with on shield i suppose.

Comment (natalieethell) [reply apps/fabric-website/src/components/App/AppState.tsx]: Oh I see, because of the `className: 'componentsPage'`?

**Comment by Jahnp** [comment]
Shoot--just realized that this breaks the UHF links to these pages, which point to the old versions of these links:
image

This issue is live in prod now. @mikewheaton / @Vitalius1 -- are either of you able to make the change in Compass?

**Comment by kenotron** [comment]
Or maybe ahem remove the top links? :)

**Comment by Jahnp** [comment]
From the UHF? Why would we remove those?

**Pull request merged**